include/arch/arm/aarch32/cortex_a_r/cmsis_ext.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief CMSIS extension
+ *
+ * This header provides CMSIS-style register access functions and macros that
+ * are not currently available in the CMSIS.
+ *
+ * NOTE: cmsis.h includes this file; do not manually include this file.
+ */
+
+#ifndef ZEPHYR_INCLUDE_ARCH_ARM_AARCH32_CORTEX_A_R_CMSIS_EXT_H_
+#define ZEPHYR_INCLUDE_ARCH_ARM_AARCH32_CORTEX_A_R_CMSIS_EXT_H_
+
+/* FSR Register Definitions */
+#define FSR_FS_BACKGROUND_FAULT		(0)
+#define FSR_FS_ALIGNMENT_FAULT		(1)
```

Comment:
@stephanosio maybe you want to fix some of this column alignments here

Reply:
It is just GitHub that is displaying the alignment incorrectly.

If you open this file on a text editor whose tab width is configured as 8, it will be properly aligned.

Reply:
https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/ae10a8c4bfb8410e7c74da103142abe18043f916/include/arch/arm/aarch32/cortex_a_r/cmsis_ext.h